hw/ip/aon_timer/dv/cov/aon_timer_unr_manually_excluded_excl.el

```diff
@@ -0,0 +1,186 @@
+INSTANCE: tb.dut.u_reg.u_reg_if.u_rsp_intg_gen
+ANNOTATION: "tl_i.d_user is hardcoded to 0x0 "
+Block 1 "461445014" "assign rsp_intg = tl_i.d_user.rsp_intg;"
+ANNOTATION: "tl_i.d_user is hardcoded to 0x0"
+Block 2 "2643129081" "assign data_intg = tl_i.d_user.data_intg;"
+CHECKSUM: "2815520955 1214062232"
+INSTANCE: tb.dut.u_reg.u_wkup_count_hi_cdc.u_arb
+ANNOTATION: "Default statement can't be hit unless some wrong value is forced in the `state_d` signal"
+Block 43 "3846199569" "gen_wr_req.state_d = StIdle;"
+CHECKSUM: "2815520955 1214062232"
+INSTANCE: tb.dut.u_reg.u_wkup_count_lo_cdc.u_arb
+ANNOTATION: "Default statement can't be hit unless some wrong value is forced in the `state_d` signal"
+Block 43 "3846199569" "gen_wr_req.state_d = StIdle;"
+CHECKSUM: "2815520955 1214062232"
+INSTANCE: tb.dut.u_reg.u_wdog_count_cdc.u_arb
+ANNOTATION: "Default statement can't be hit unless some wrong value is forced in the `state_d` signal"
+Block 43 "3846199569" "gen_wr_req.state_d = StIdle;"
+CHECKSUM: "2815520955 1214062232"
+INSTANCE: tb.dut.u_reg.u_wkup_cause_cdc.u_arb
+ANNOTATION: "Default statement can't be hit unless some wrong value is forced in the `state_d` signal"
+Block 43 "3846199569" "gen_wr_req.state_d = StIdle;"
+CHECKSUM: "2815520955 4051034043"
+INSTANCE: tb.dut.u_reg.u_wkup_count_hi_cdc.u_arb
+ANNOTATION: "Default statement can't be hit unless some wrong value is forced in the `state_d` signal"
+Branch 4 "3547459906" "gen_wr_req.state_q" (6) "gen_wr_req.state_q default,-,-,-,-"
+CHECKSUM: "2815520955 4051034043"
+INSTANCE: tb.dut.u_reg.u_wkup_count_lo_cdc.u_arb
+ANNOTATION: "Default statement can't be hit unless some wrong value is forced in the `state_d` signal"
+Branch 4 "3547459906" "gen_wr_req.state_q" (6) "gen_wr_req.state_q default,-,-,-,-"
+CHECKSUM: "2815520955 4051034043"
+INSTANCE: tb.dut.u_reg.u_wdog_count_cdc.u_arb
+ANNOTATION: "Default statement can't be hit unless some wrong value is forced in the `state_d` signal"
+Branch 4 "3547459906" "gen_wr_req.state_q" (6) "gen_wr_req.state_q default,-,-,-,-"
+CHECKSUM: "2815520955 4051034043"
+INSTANCE: tb.dut.u_reg.u_wkup_cause_cdc.u_arb
+ANNOTATION: "Default statement can't be hit unless some wrong value is forced in the `state_d` signal"
+Branch 4 "3547459906" "gen_wr_req.state_q" (6) "gen_wr_req.state_q default,-,-,-,-"
+CHECKSUM: "530940107 4102526809"
+INSTANCE: tb.dut
+ANNOTATION: "intr_test_flds_we[0/1] is asserted at the same time whenever the intr_test register is written, so the combination of only one Qe being set is not possible."
+Condition 4 "594611319" "(reg2hw.intr_test.wkup_timer_expired.qe | reg2hw.intr_test.wdog_timer_bark.qe) 1 -1" (3 "10")
+Condition 4 "594611319" "(reg2hw.intr_test.wkup_timer_expired.qe | reg2hw.intr_test.wdog_timer_bark.qe) 1 -1" (2 "01")
+CHECKSUM: "1125143680 279071062"
+INSTANCE: tb.dut.u_reg.u_wkup_count_hi_cdc
+ANNOTATION: "src_ack is only 1 cycle pulse, and is an acknowledgment to a request from a read/write. 
```

Comment:
This is another example where I think an RTL change is the right move. As your note says, there's even an assertion that checks the behaviour won't change :-)